rum-injection-installer/installer/nginx_configurator.go

```diff
@@ -0,0 +1,272 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+type NginxConfigurator struct {
+	Version     string
+	ModulesPath string
+}
+
+// Runs 'nginx -v' and looks for the nginx version.
+// Checks that the version is higher than a min version
+func (n *NginxConfigurator) Verify() error {
```

Comment:
**[suggestion]** the function name is a bit too vague. What is being verified? May I suggest `VerifyRequirement`?